features/manage-position/Create.tsx

```diff
@@ -127,7 +127,8 @@ const Create = () => {
     const balanceBelowCollateralToDeposit = balance < collateralToDeposit;
     const needAllowance =
       collAllowance !== "Infinity" && collAllowance < collateralToDeposit;
-    const resultantTokensBelowMin = resultantTokens < minSponsorTokensFromWei;
+    const resultantTokensBelowMin =
+      resultantTokens < minSponsorTokensFromWei && resultantTokens != 0;
```

Comment:
This is a good add but nit: add a second "=" . Its consistent with the rest of the code and we should avoid implicitly cast any types, which != does.

https://howtodoinjava.com/javascript/javascript-equality-vs-identity-operators/